proc_noise.py

```diff
@@ -69,13 +68,10 @@
     fl.next('noise -- semilog', legend=True)
     fl.plot(noise['t2':frq_slice], plottype='semilogy',alpha=0.05, color=c,
             human_units=False)
-    #noise.convolve('t2',500) # convolution is just to provide a running
-    # average, to make it clear where the average noise baseline is at (remember
-    # we are doing this *after* taking the abs)
```

Comment:
so, this seems bad -- why did this get commented out in the first place?

Reply:
I should elaborate -- with the noise processing, I would argue that taking the convolution after the abs is actually essential to getting a good noise PSD within a reasonable number of scans.

we are deliberately sacrificing frequency resolution (don't care about the frequency resolution of our noise very much!) for a good SNR curve (odd to use that expression, but there is an SNR to the noise density measurement itself).